x-pack/plugins/security_solution/public/notes/api/api.ts

```diff
@@ -30,32 +29,20 @@ export const createNote = async ({ note }: { note: BareNote }) => {
   }
 };
 
-// TODO point to the correct API when it is available
 /**
  * Fetches all the notes for an array of document ids
  */
 export const fetchNotesByDocumentIds = async (documentIds: string[]) => {
-  const response = {
-    totalCount: 1,
-    notes: generateNoteMock(documentIds),
-  };
-  return response.notes;
+  const response = await KibanaServices.get().http.get<{ notes: Note[]; totalCount: number }>(
+    NOTE_URL,
+    {
+      query: { alertIds: documentIds },
```

Comment:
PR looks good! Just minor change...probably best to keep this as `documentIds` since it adds notes for events and alerts..